### PR TITLE
[stable/sonatype-nexus] Add openshift routes

### PR DESCRIPTION
**Pull request by hamza3202**
**Problem:**
No resource to create openshift routes.

**Solution**
Add routes for both services.

**What this PR does?**
Adds new routes for both docker and nexus(proxy) service.

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/sonatype-nexus/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: sonatype-nexus
-version: 1.17.0
+version: 1.17.1
```
Comment (rjkernick) [inline stable/sonatype-nexus/Chart.yaml]: Please bump the minor version as this add features

**Comment by hamza3202** [comment]
@rjkernick any updates on this?

File: stable/sonatype-nexus/README.md [hunk 2]
```diff
@@ -144,6 +148,13 @@ The following table lists the configurable parameters of the Nexus chart and the
...
 | `service.port`                              | Port for exposing service        | `nil`                                      |
+| `route.enabled`         | Set to true to create route for additional service | false |
+| `route.name`            | Name of route                                      | false |
+| `route.portName`        | Target port name of service                        | false |
+| `route.labels`          | Labels to be added to route                        | false |
+| `route.annotations`     | Annotations to be added to route                   | false |
+| `route.path`            | Host name of Route                                 | false |
+
 
 If `nexusProxy.env.cloudIamAuthEnabled` is set to `true` the following variables need to be configured
```
Comment (rjkernick) [inline stable/sonatype-nexus/README.md:159]: Maybe add a section on what an example set of overrides for openshift look like.

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by hamza3202** [comment]
@rjkernick any updates on this?

**Pull request merged**